transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/AlarmManagerScheduler.java

```diff
@@ -78,6 +81,10 @@ public void schedule(TransportContext transportContext, int attemptNumber) {
     intentDataBuilder.appendQueryParameter(BACKEND_NAME, transportContext.getBackendName());
     intentDataBuilder.appendQueryParameter(
         EVENT_PRIORITY, String.valueOf(transportContext.getPriority().ordinal()));
+    if (transportContext.getExtras() != null) {
+      intentDataBuilder.appendQueryParameter(
+          EXTRAS, encodeToString(transportContext.getExtras(), DEFAULT));
```

Comment:
I think it would be more readable if you did not add a wildcard import above, as, by just looking at his line, it is not obvious what encoding is used and what `DEFAULT` means, would it make sense to change to `Base64.encodeToString()`?